clang/lib/Basic/OffloadArch.cpp

```diff
@@ -0,0 +1,126 @@
+#include "clang/Basic/OffloadArch.h"
+
+#include "llvm/ADT/StringRef.h"
+
+#include <algorithm>
+
+namespace clang {
+
+namespace {
+struct OffloadArchToStringMap {
+  OffloadArch arch;
+  const char *arch_name;
+  const char *virtual_arch_name;
```

Comment:
While we're here, can we update these to the LLVM style?
```suggestion
  OffloadArch Arch
  const char *ArchName;
  const char *VirtualArchName;
```

Reply:
Good catch. [d43ae13](https://github.com/llvm/llvm-project/pull/137070/commits/d43ae1368c8ac5375fa63498697083fe02fe1941)